labels/en-custom.json

```diff
@@ -8,5 +8,7 @@
 	"gmd:DQ_LogicalConsistency": "Data Quality Reports (Logical Consistency)",
 	"gmd:DQ_FormatConsistency": "Data Quality Reports (Format Consistency)",
 	"gmd:DQ_ConceptualConsistency": "Data Quality Reports (Conceptual Consistency)",
-	"gmd:DQ_DomainConsistency": "Data Quality Reports (Domain Consistency and INSPIRE)"
+	"gmd:DQ_DomainConsistency": "Data Quality Reports (Domain Consistency and INSPIRE)",
+	"gmd:LI_Lineage": "Data Quality Reports (Lineage)",
```

Comment:
I think I would call this something like: Lineage with process step to distinguish it from the other data quality reports